contrib/kind.sh

```diff
@@ -27,83 +27,86 @@ usage()
 {
     echo "usage: kind.sh [[[-cf|--config-file <file>] [-kt|keep-taint] [-ha|--ha-enabled]"
     echo "                 [-ho|--hybrid-enabled] [-ii|--install-ingress] [-n4|--no-ipv4]"
-    echo "                 [-i6|--ipv6] [-wk|--num-workers <num>]"
+    echo "                 [-i6|--ipv6] [-wk|--num-workers <num>] [ds|--disable-snat-multiples-gws]"
```

Comment:
`-` before `ds`, and `multiple`, not `multiples`

Reply:
done